src/main.cpp

```diff
@@ -26,8 +26,12 @@ int main(int argc, char *argv[])
   QSettings *configini = main_app.configini;
 
   QPluginLoader apngPlugin("qapng");
+#ifdef STATIC_LIB
+  Q_IMPORT_PLUGIN(ApngImagePlugin)
```

Comment:
From my understanding, this goes immediately after the includes, not inside `main()`.